src/Pos/Wallet/Light/Launcher/Runner.hs

```diff
@@ -121,3 +124,12 @@ runRawStaticPeersWallet transport peers WalletParams {..}
             wpDbPath
     closeDB = closeState
 {-# NOINLINE runRawStaticPeersWallet #-}
+
+runServer_
```

Comment:
What does this function do? Can you add a comment? `const pass` makes it look suspicious.